SCOPE.md

```diff
@@ -5,7 +5,7 @@ These following criteria will be used when reviewing pull requests, features, an
 
 ### Components
 
-Components should not have tight dependencies on each other so that they are able to be used independently.
+Components should not have tight dependencies on each other so that they are able to be use independently.
```

Comment:
"used" is the correct word here

Reply:
I understand something like to be able to do without `d`

Reply:
I don't understand what you said.

"to be use" is not a grammatical phrase in English.